tests/chains/Bitcoin/InputSelectorTests.cpp

```diff
@@ -134,14 +136,28 @@ TEST(BitcoinInputSelector, SelectOneInsufficientHigher) {
     auto selector = InputSelector<UTXO>(utxos);
     auto selected = selector.select(99'900, 1);
 
-    EXPECT_TRUE(verifySelectedUTXOs(selected, {}));
+    // `InputSelector` returns the entire list of UTXOs even if they are not enough.
+    // That's because `InputSelector` has a rough segwit fee estimation algorithm, and the UTXOs can actually be enough.
+    EXPECT_TRUE(verifySelectedUTXOs(selected, {100'000}));
+}
+
+TEST(BitcoinInputSelector, SelectOneInsufficientHigherFilterDust) {
+    auto utxos = buildTestUTXOs({22, 100'000, 40});
+
+    auto selector = InputSelector<UTXO>(utxos);
+    auto selected = selector.select(99'900, 1);
+
+    // `InputSelector` returns the entire list of UTXOs even if they are not enough.
```

Comment:
This comment is incorrect (?)

Reply:
Yes, that's correct, `InputSelector` returns all UTXOs (except less Dust Threshold) in this test